### PR TITLE
routes for seed file

### PR DESCRIPTION
**Pull request by pamtimu**

### CODE REVIEW

File: app/controllers/routes_controller.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 class RoutesController < ApplicationController
+
+  def index
+    @routes = Route.all
+  end
+
+  def show
+    @route = Route.find(params[:id])
+  end
+
```
Comment (carbonarichristian) [inline app/controllers/routes_controller.rb:10]: Aren't we adding the strong params?

Comment (pamtimu) [reply app/controllers/routes_controller.rb:10]: yes great catch! ill change it

File: app/controllers/routes_controller.rb [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
   end
 
   def show
-    @route = Route.find(params[:id])
   end
 
+  private
+
+  def route_params
+    params.require(:route).permit(:name, :description)
```
Comment (carbonarichristian) [inline app/controllers/routes_controller.rb:14]: Should we change the permissions to address and distance?

**Comment by carbonarichristian** [comment]
Looks good

**Pull request merged**